src/GafferScene/IECoreGLPreview/Renderer.cpp

```diff
@@ -366,6 +405,27 @@ class OpenGLObject : public IECoreScenePreview::Renderer::ObjectInterface
 			{
 				glPopMatrix();
 			}
+
+			// Local-scale-free visualisations, that use the attribute driven
+			// visualiserScale for size adjustments.
+			if( m_attributes->ornamentScale() > 0 )
+			{
+				if( auto v = visualisation( *m_attributes, VisualisationType::Ornament ) )
+				{
+					if( haveTransform )
```

Comment:
This check is breaking visualiser scale for lights at the origin.

Reply:
Good spot, thanks. https://github.com/GafferHQ/gaffer/pull/3514/commits/91a60b86500ee0271c7bc75f8137350470d3b1c7